Sources/Algorithms/Combinations.swift

```diff
@@ -2,7 +2,7 @@
 //
 // This source file is part of the Swift Algorithms open source project
 //
-// Copyright (c) 2020 Apple Inc. and the Swift project authors
+// Copyright (c) 2021 Apple Inc. and the Swift project authors
```

Comment:
This should be a range now:
```suggestion
// Copyright (c) 2020-2021 Apple Inc. and the Swift project authors
```